R/plot_map.R

```diff
@@ -97,14 +94,6 @@
 #' # Voronoi plots
 #' smpl <- sample(faux_cell, 300)
 #' plotmap(smpl, region ~ x * y)
-#' if (require(tripack)) {
```

Comment:
btw, we could leave the `plotvoronoi()` example without further arguments in here since we anyways import from **latticeExta**.

Reply:
The example is now present again.